{
  "comments": [
    {
      "key": {
        "uuid": "fafc7978_e1ba7009",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 7,
      "author": {
        "id": 2046
      },
      "writtenOn": "2017-01-23T15:10:38Z",
      "side": 1,
      "message": "Well, it is still there, just demoted into a failure message. But this is consistent with commit messages of similar Changes.",
      "range": {
        "startLine": 7,
        "startChar": 0,
        "endLine": 7,
        "endChar": 26
      },
      "revId": "f8d2f282019fcbd4d6a2a4edeceb5fa6f36a87e4",
      "serverId": "7fc14799-209e-464c-9743-7a06c2c21a81",
      "unresolved": false
    }
  ]
}